updates to readme (#97)
kartoza/stream_feature_extractor: README.md

See the LICENSE file included with the plugin (and in this repository) for
more information about this license.

# Installation of the plugin

> The easiest approach will be to install the plugin using the QGIS plugin manager:

Reviewer: ```suggestion
Open the QGIS plugin manager in order to install the plugin:
```

7. 3.22; and
8. latest version.

## Local testing

> To perform local testing the 'run-docker-tests.sh' can be used. The shell script can be used as follows:

Reviewer: We need to outline that this script can run only under linux environments.

1. Open your terminal/console;
2. Go to the root directory of the plugin;
3. Type "./run-docker-test.sh" and press enter;
4. The result should be as follows:

![segment_center](https://github.com/kartoza/stream_feature_extractor/blob/develop/help/source/examples/local_testing.png)

> 5. If there is any errors the user will need to investigate.

Reviewer: We should remove this line, and instead explain what is need for the tests to run successfully.
